### PR TITLE
Add contracts table and subgraph table to README

### PR DESCRIPTION
**Pull request by pimato**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,26 @@ Create a `.env` file. See `.env.example` for environment variables which may be
...
+| Reality.eth Oracle Adapters (Scalar) | [0xb97FCb6adf4c4aF9981932a004e6CC47173d0Bfc](https://blockscout.com/poa/xdai/address/0xb97FCb6adf4c4aF9981932a004e6CC47173d0Bfc/contracts) | [0x1D369EEC97cF2E62c8DBB804b3998Bf15bcb67dB](https://blockscout.com/poa/sokol/address/0x1D369EEC97cF2E62c8DBB804b3998Bf15bcb67dB/contracts) | -     |
+| CPKFactory | [0xfC7577774887aAE7bAcdf0Fc8ce041DA0b3200f7](https://blockscout.com/poa/xdai/address/0xfC7577774887aAE7bAcdf0Fc8ce041DA0b3200f7/contracts) | [0xaaF0CCef0C0C355Ee764B3d36bcCF257C527269B](https://blockscout.com/poa/sokol/address/0xaaF0CCef0C0C355Ee764B3d36bcCF257C527269B/contracts) | -     |
+| Uniswap V2 Router | [0x1C232F01118CB8B424793ae03F870aa7D0ac7f77](https://blockscout.com/poa/xdai/address/0x1C232F01118CB8B424793ae03F870aa7D0ac7f77/contracts) | [0x5948f454fceF54e81757e96f7ebb2b91A064771c](0x5948f454fceF54e81757e96f7ebb2b91A064771c) | -     |
```
Comment (KadenZipfel) [inline README.md:32]: Looks like an issue with the second link here

**Review by KadenZipfel — CHANGES_REQUESTED** [review]
Should we also include mainnet and rinkeby contracts?

File: README.md [hunk 2]
```diff
@@ -16,3 +16,26 @@ Create a `.env` file. See `.env.example` for environment variables which may be
...
 
 The `build` directory in the `app` directory will now contain the build to be served.
+
+
+## Contracts
+
+| Contract | xDai | Sokol | Arbitrum |
+| -------- | -------- | -------- | -------- |
+| [ConditionalTokens](https://github.com/gnosis/conditional-tokens-contracts) | [0xCeAfDD6bc0bEF976fdCd1112955828E00543c0Ce](https://blockscout.com/poa/xdai/address/0xCeAfDD6bc0bEF976fdCd1112955828E00543c0Ce/read-contract)     | [0x0Db8C35045a830DC7F2A4dd87ef90e7A9Cd0534f](https://blockscout.com/poa/sokol/address/0x0Db8C35045a830DC7F2A4dd87ef90e7A9Cd0534f/contracts)     | -     |
+| FPMMDeterministicFactory | [0x9083A2B699c0a4AD06F63580BDE2635d26a3eeF0](https://blockscout.com/poa/xdai/address/0x9083A2B699c0a4AD06F63580BDE2635d26a3eeF0/contracts) | [0x2fb8cc057946DCFA32D8eA8115A1Dd630f6efea5](https://blockscout.com/poa/sokol/address/0x2fb8cc057946DCFA32D8eA8115A1Dd630f6efea5/contracts) | -     |
```
Comment (hexyls) [inline README.md:26]: We can add a link for the market maker contracts: https://github.com/gnosis/conditional-tokens-market-makers

**Review by hexyls — APPROVED** [review]
🙌

File: README.md [hunk 3]
```diff
@@ -16,3 +16,79 @@ Create a `.env` file. See `.env.example` for environment variables which may be
...
+| Uniswap V2 Router | [0x1C232F01118CB8B424793ae03F870aa7D0ac7f77](https://blockscout.com/poa/xdai/address/0x1C232F01118CB8B424793ae03F870aa7D0ac7f77/contracts) |
+| Uniswap V2 Factory | [0xA818b4F111Ccac7AA31D0BCc0806d64F2E0737D7](https://blockscout.com/poa/xdai/address/0xA818b4F111Ccac7AA31D0BCc0806d64F2E0737D7/contracts) |
+| Wrapped1155Factory | [0xDE6943f3717738038159a406FF157d4eb3238c1B](https://blockscout.com/poa/xdai/address/0xDE6943f3717738038159a406FF157d4eb3238c1B/contracts) |
+
+
+## Sokol Network
+
+| Contract | Address |
+| -------- | -------- |
+| [ConditionalTokens](https://github.com/gnosis/conditional-tokens-contracts) | [0x0Db8C35045a830DC7F2A4dd87ef90e7A9Cd0534f](https://etherscan.io/address/0xC59b0e4De5F1248C1140964E0fF287B192407E0C#code)     |
```
Comment (KadenZipfel) [inline README.md:77]: Looks like a mainnet address slipped in here

**Review by KadenZipfel — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (pimato) [reply README.md]: fixed

File: README.md [hunk 4]
```diff
@@ -16,3 +16,79 @@ Create a `.env` file. See `.env.example` for environment variables which may be
...
+| Uniswap V2 Factory | [0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f](https://rinkeby.etherscan.io/address/0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f#code) |
+| WETH9 | [0xc778417E063141139Fce010982780140Aa0cD5Ab](https://rinkeby.etherscan.io/address/0xc778417E063141139Fce010982780140Aa0cD5Ab#code) |
+| GelatoCore | [0x733aDEf4f8346FD96107d8d6605eA9ab5645d632](https://rinkeby.etherscan.io/address/0x733aDEf4f8346FD96107d8d6605eA9ab5645d632#code) |
+
+
+## xDai Network
+
+| Contract | Address |
+| -------- | -------- |
+| [ConditionalTokens](https://github.com/gnosis/conditional-tokens-contracts) | [0xCeAfDD6bc0bEF976fdCd1112955828E00543c0Ce](https://blockscout.com/poa/xdai/address/0xCeAfDD6bc0bEF976fdCd1112955828E00543c0Ce/read-contract)     |
```
Comment (Mi-Lan) [inline README.md:61]: For the sake of consistency i would add here so that it point to /contracts like rest

File: README.md [hunk 5]
```diff
@@ -16,3 +16,79 @@ Create a `.env` file. See `.env.example` for environment variables which may be
...
+| Reality.eth Oracle Adapters | [0xa57EBD93faa73b3491aAe396557D6ceC24fC6984](https://blockscout.com/poa/sokol/address/0xa57EBD93faa73b3491aAe396557D6ceC24fC6984/contracts) |
+| Reality.eth Oracle Adapters (Scalar) | [0x1D369EEC97cF2E62c8DBB804b3998Bf15bcb67dB](https://blockscout.com/poa/sokol/address/0x1D369EEC97cF2E62c8DBB804b3998Bf15bcb67dB/contracts) |
+| CPKFactory | [0xaaF0CCef0C0C355Ee764B3d36bcCF257C527269B](https://blockscout.com/poa/sokol/address/0xaaF0CCef0C0C355Ee764B3d36bcCF257C527269B/contracts) |
+| Uniswap V2 Router | [0x5948f454fceF54e81757e96f7ebb2b91A064771c](https://blockscout.com/poa/sokol/address/0x5948f454fceF54e81757e96f7ebb2b91A064771c/contracts) |
+| Uniswap V2 Factory | [0x985B5011c850C27ee1cE0a0982B8E9c230596960](https://blockscout.com/poa/sokol/address/0x985B5011c850C27ee1cE0a0982B8E9c230596960/contracts) |
+| Wrapped1155Factory | [0xDE6943f3717738038159a406FF157d4eb3238c1B](https://blockscout.com/poa/sokol/address/0xDE6943f3717738038159a406FF157d4eb3238c1B/transactions) |
```
Comment (Mi-Lan) [inline README.md:86]: Here as well...

**Review by Mi-Lan — CHANGES_REQUESTED** [review]
Also i think it would be good to fetch these contract addresses dynamically from the networks since it will be much easier to maintain...This could be useful for dynamic md file maybe?https://github.com/marketplace/actions/dynamic-readme @KadenZipfel @pimato ?

**Comment by KadenZipfel** [comment]
> Also i think it would be good to fetch these contract addresses dynamically from the networks since it will be much easier to maintain...This could be useful for dynamic md file maybe?https://github.com/marketplace/actions/dynamic-readme @KadenZipfel @pimato ?

This is a really good idea, it's way too easy to forget to update the readme.

**Comment by pimato** [comment]
> > Also i think it would be good to fetch these contract addresses dynamically from the networks since it will be much easier to maintain...This could be useful for dynamic md file maybe?https://github.com/marketplace/actions/dynamic-readme @KadenZipfel @pimato ?
> 
> This is a really good idea, it's way too easy to forget to update the readme.

Just created an issue for it: https://github.com/protofire/omen-exchange/issues/1634

**Review by KadenZipfel — APPROVED** [review]

**Review by pRoy24 — APPROVED** [review]
&#128640;

**Pull request merged**